### PR TITLE
Improvements to windows build instructions

### PR DESCRIPTION
**Pull request by jtattermusch**
- some readability improvements

### CODE REVIEW

**Comment by jtattermusch** [comment]
CC @ctiller  @SurferJeffAtGoogle

File: INSTALL.md [hunk 1]
```diff
@@ -94,42 +94,51 @@ on experience with the tools involved.
...
+- Install [Go](https://golang.org/dl/) (`choco install golang`) - *required by boringssl*
+- Install [yasm](http://yasm.tortall.net/) and add it to `PATH` (`choco install yasm`) - *required by boringssl*
+- (Optional) Install [Ninja](https://ninja-build.org/) (`choco install ninja`)
+
+#### Clone grpc sources including submodules
+Before building, you need to clone the gRPC github repository and download submodules containing source code 
+for gRPC's dependencies (that's done by the `submodule` command).
 ```
-> powershell git clone --recursive -b ((New-Object System.Net.WebClient).DownloadString(\"https://grpc.io/release\").Trim()) https://github.com/grpc/grpc
+> @rem You can also do just "git clone -b THE_BRANCH_YOU_WANT https://github.com/grpc/grpc"
```
Comment (mehrdada) [inline INSTALL.md]: I feel like omitting `--recursive` in favor of explicit submodule checkout is a regression. People forget to run the submodule init and file build failed bugs

Comment (jtattermusch) [reply INSTALL.md]: Ok, I might change it back. The difference is that we only need recursive with limited depth (e.g. not necessary to fetch protobuf's submodules). I actually might add --recurse-depth 2

Comment (mehrdada) [reply INSTALL.md]: `--recursive` is not perceptibly slower and it does not really hurt IMO. I think a shorter command line might be better but your call.

**Pull request merged**